ctutil/sctcheck/sctcheck.go

```diff
@@ -113,15 +114,19 @@ func checkChain(ctx context.Context, lf logInfoFactory, chain []*x509.Certificat
 	}
 
 	var issuer *x509.Certificate
-	if len(chain) < 2 {
+	for _, c := range chain[1:] {
+		if bytes.Equal(c.SubjectKeyId, leaf.SubjectKeyId) {
+			issuer = c
+		}
```

Comment:
I wonder if it's worth just logging when an out-of-order chain is detected, as a sort of "lint" thing?

Reply:
As the out-of-order chain is not supported in TLS 1.2, I'm not sure if anyone is using this tool only for TLS 1.2. A warning log can probably help them.

Reply:
LGTM

Reply:
Great, this can save an access request than #1111.

Reply:
Thanks for the review.